src/main/java/org/jenkinsci/plugins/workflow/support/steps/input/InputStepExecution.java

```diff
@@ -93,9 +93,13 @@ public boolean start() throws Exception {
         InputStep inputStep = this.getInput();
         Jenkins.get().getExtensionList(InputExtension.class).forEach(input -> {
             String name = input.getName();
+            if(StringUtils.isEmpty(name)) {
+                // make sure we have a name
+                name = input.getClass().getSimpleName();
```

Comment:
Seems unnecessary, I was more thinking along the lines whether we can just not require having this method.

If we want to have a default, should probably be a default interface method instead.